### ISSUE TITLE
Fails on nullish coalescing operator

### ISSUE THREAD

**Issue by TheUnlocked** [mozilla/addons-linter#3154]
### Describe the problem and steps to reproduce it:

Use the nullish coalescing operator `??` in a file and try to lint it.

Minimal repro:
```js
let x = undefined ?? "";
```

### What happened?

The linter reported a generic syntax error.

### What did you expect to happen?

No error.

### Anything else we should know?

The linter reports the error as occurring at line 1, col 1 despite the error being caused by the operator.

**Comment by Rob--W** [comment]
Duplicate of #2978

**Comment by Rob--W** [comment]
Sorry, edited the wrong issue. Still related though: this error is because the new syntax is not yet supported by eslint.

**Comment by willdurand** [comment]
This is probably fixed by https://github.com/mozilla/addons-linter/pull/3161
